src/main/java/uk/gov/companieshouse/api/strikeoffobjections/file/FileTransferApiClientResponse.java

```diff
@@ -0,0 +1,35 @@
+package uk.gov.companieshouse.api.strikeoffobjections.file;
+
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.HttpStatus;
+
+public class FileTransferApiClientResponse {
+
+    private String fileId;
+    private HttpStatus httpStatus;
+    private HttpHeaders headers;
```

Comment:
httpHeaders for this member variable name, for consistency with httpStatus.

Reply:
Renamed